### PR TITLE
Styling fellow directory component

### PR DESCRIPTION
**Pull request by youriwims**
Related PRs/issues #2669, #2684

## Checklist

- [x] Add social icons
- [x] Update Fellow aspect ratio to 1:1
- [x] Style directory component

### CODE REVIEW

**Comment by Pomax** [comment]
Nice, glad to see the CSS grid solution behaving properly on the various sizes!

**Comment by Pomax** [comment]
@kristinashu this should be ready for design review

**Comment by kristinashu** [comment]
Amazing, this is looking fantastic! Thank you all for the hard work. 

This is all I could find:
- [ ] Update social icon hover state to #0d10bf (sorry but since opening the ticket Mavis has updated our blue link color to this more purple version)
- [ ] Mobile: images should be 1:1 ratio

Screen cap from PR:
screen shot 2019-02-21 at 2 46 25 pm

- [ ] Hover state for names are odd when on two lines, is there a way to fix this?
Screen cap from PR:
screen shot 2019-02-21 at 2 46 36 pm

**Comment by kristinashu** [comment]
I've updated my feedback above with a little more context.

**Comment by youriwims** [comment]
Thanks @kristinashu !

File: network-api/networkapi/settings.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
     DEBUG=(bool, False),
     DJANGO_LOG_LEVEL=(str, 'INFO'),
     DOMAIN_REDIRECT_MIDDLWARE_ENABLED=(bool, False),
-    ENABLE_PROFILE_DIRECTORY_BLOCK=(bool, False),
+    ENABLE_PROFILE_DIRECTORY_BLOCK=(bool, True),
```
Comment (Pomax) [inline network-api/networkapi/settings.py]: no, let's keep this `False`. We'll remove this flag in a follow-up https://github.com/mozilla/foundation.mozilla.org/issues/2706

Comment (youriwims) [reply network-api/networkapi/settings.py]: Done.

**Comment by Pomax** [comment]
@kristinashu this is ready for (re)review =P

**Comment by kristinashu** [comment]
w00t looks great! 

I notice that at mobile, some of the spacing between individual profile cards is pretty inconsistent but that can be done as a follow up! I'll open a new ticket.

image

**Pull request merged**